src/sentry/models/releasethreshold.py

```diff
@@ -0,0 +1,35 @@
+from django.db import models
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import FlexibleForeignKey, Model, region_silo_only_model
+
+
+@region_silo_only_model
+class ReleaseThreshold(Model):
+    __relocation_scope__ = RelocationScope.Excluded
+
+    class ReleaseThresholdType(models.TextChoices):
+        TOTAL_ERROR_COUNT = "TEC", _("Total Error Count")
```

Comment:
🤔 hmm...

I wonder if rather than codifying this within the model, we simply make this an enum in a constants file?

for _now_ we have a predefined set of thresholds you can choose from.
In the future i imagine we'd either expand this set, or even allow a simple string of sorts to determine where to look for our stats.

Reply:
actually.
I guess modifying this wouldn't require a migration of any sort... 🤔 

still feels like it can go outside of the model definition though 🤷

Reply:
i like enum so you can reference it in the incoming serializer in the API too

Reply:
We have a few enums that also implement an `as_choices` method that allows them to play nicely with django's ChoiceFields. `MonitorStatus` is an example of that.

Reply:
ok fixed the enum